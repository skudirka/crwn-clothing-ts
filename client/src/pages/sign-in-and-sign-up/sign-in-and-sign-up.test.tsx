import {shallow} from 'enzyme';
import React from 'react';

import SignInAndSignUpPage from './sign-in-and-sign-up.component';

describe('SignInAndSignUpPage page', () => {
    it('SignInAndSignUpPage component should render', () => {
        expect(shallow(<SignInAndSignUpPage />)).toMatchSnapshot();
    })
});