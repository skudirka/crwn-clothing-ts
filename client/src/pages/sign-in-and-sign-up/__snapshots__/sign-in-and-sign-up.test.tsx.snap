// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInAndSignUpPage page SignInAndSignUpPage component should render 1`] = `ShallowWrapper {}`;
